docs/contribute.rst

```diff
@@ -82,11 +88,11 @@ the nature of your changes, then you should update the output used by the
 functional tests. To do so, **make sure you have both** ``en_EN.utf8`` **and**
 ``fr_FR.utf8`` **locales installed**, and then run the following two commands::
 
-    $ LC_ALL=en_US.utf8 pelican -o pelican/tests/output/custom/ \
+    LC_ALL=en_US.utf8 pelican -o pelican/tests/output/custom/ \
         -s samples/pelican.conf.py samples/content/
-    $ LC_ALL=fr_FR.utf8 pelican -o pelican/tests/output/custom_locale/ \
+    LC_ALL=fr_FR.utf8 pelican -o pelican/tests/output/custom_locale/ \
         -s samples/pelican.conf_FR.py samples/content/
-    $ LC_ALL=en_US.utf8 pelican -o pelican/tests/output/basic/ \
+    LC_ALL=en_US.utf8 pelican -o pelican/tests/output/basic/ \
         samples/content/
 
```

Comment:
Maybe we can make an invoke task for this?

Reply:
Good idea. I added an Invoke task for updating functional test output in b7368f91.